frame/auction.ts

```diff
@@ -20,7 +20,12 @@ import { FetchJsonStatus, tryFetchJson } from "./fetch";
  * validates that it is a JSON object. For now, the data is simply thrown away
  * after that initial validation, because the simple scoring algorithm currently
  * used has no use for scoring signals, but when more algorithms are supported,
- * this data will be passed to them.
+ * this data will be passed to them. No request is made if no ads are available.
+ *
+ * Two query parameters are appended to `trustedScoringSignalsUrl`: `hostname`,
```

Comment:
I don't see hostname appended to `trustedScoringSignalsUrl` in the spec?  Only `trustedBiddingSignalsUrl`: https://github.com/WICG/turtledove/blob/main/FLEDGE.md#31-fetching-real-time-data-from-a-trusted-server

Since `trustedScoringSignalsUrl` comes from the call to `runAdAuction` it seems to me that the seller could include the hostname or anything else in the URL.

Reply:
The explainer's wording is ambiguous, and Chrome hasn't implemented this yet so I can't look up what their code does, but you make a good point about this not being necessary in principle. I'll drop it.